### PR TITLE
Implemented jobs-listings page with grid layout

### PR DESCRIPTION
**Pull request by Ekman1991**

### CODE REVIEW

**Comment by sirmackan** [comment]
👍

File: app/app.component.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
     </nav>
   </div>
 </div>
-<section style="margin-top: 54px;">
+<section class="container-fluid" style="margin-top: 54px;">
```
Comment (buren) [inline app/app.component.html:8]: Inline styling?

File: app/components/job-list-item/job-list-item.component.html [hunk 2]
```diff
@@ -1,40 +1,24 @@
-<div class="job-list-container table clickable" (click)="onListItemClick()">
-  <div class="table-row">
-    <div class="table-cell min-width-column company-logo-container">
-      <img  class="company-logo rounded-company-logo rounded-image no-select" [src]="(job.company.companyLogo && job.company.companyLogo.imageUrlSmall) || 'assets/images/placeholder-logo.png'">
+<div class="job-list-container" (click)="onListItemClick()">
+  <p style="font-weight: bold;">{{job.name}}</p>
```
Comment (buren) [inline app/components/job-list-item/job-list-item.component.html]: Inline styling?

File: app/views/jobs/jobs.component.html [hunk 3]
```diff
@@ -1,5 +1,36 @@
...
+    </h2>
+  </div>
+
+  <div class="col-md-6">
+    <!-- TODO: Implemente this -->
+  </div>
+</div>
+
+
+<div [hidden]="true" class="row" style="background: green; height: 100px;">
```
Comment (buren) [inline app/views/jobs/jobs.component.html:14]: Inline styling?

File: app/views/jobs/jobs.component.html [hunk 4]
```diff
@@ -1,5 +1,36 @@
...
+  </div>
+  <div class="forward-button col-xs-4">
+    <i class="fa fa-angle-double-right" aria-hidden="true"></i>
+  </div>
+</div>
+
+
+<div class="job-list-container row">
+  <div>
+    <job-list-item style="display: block;" *ngFor="let job of jobs" [job]="job"></job-list-item>
```
Comment (buren) [inline app/views/jobs/jobs.component.html:34]: Inline styling?

File: app/app.component.html [hunk 5]
```diff
@@ -5,7 +5,7 @@
...
   <router-outlet></router-outlet>
 </section>
 <footer style="background: yellow; height: 100px;">
```
Comment (buren) [inline app/app.component.html:11]: Inline styling?

File: app/app.component.html (hunk 1 above)
Comment (Ekman1991) [reply app/app.component.html:8]: Will be fixed in other merge

File: app/app.component.html (hunk 5 above)
Comment (Ekman1991) [reply app/app.component.html:11]: Will be fixed in other merge

File: app/views/jobs/jobs.component.html (hunk 3 above)
Comment (Ekman1991) [reply app/views/jobs/jobs.component.html:14]: This is a TODO, will need to be styled when implemented

File: app/components/job-list-item/job-list-item.component.html (hunk 2 above)
Comment (Ekman1991) [reply app/components/job-list-item/job-list-item.component.html]: Fixed in 79f0dc1

**Pull request merged**